docs/DeveloperGuide.md

```diff
@@ -324,42 +306,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
    Use case ends.
 
-**Extensions**
-
-    1a. Invalid keyword entered.
-        1a1. ExpenseExpert requests to check the keyword and enter command again.
-        Use case ends.
-
 **Use case: UC07 Set Monthly Budget**
 
 **MSS**
 1. User requests to set a monthly budget.
-2. Expense Expert sets a monthly budget for the user.
+2. User supplies details of monthly budget.
+3. Expense Expert sets a monthly budget for the user.
 
    Use case ends.
 
 **Extensions**
 
-    1a. Invalid budget entered.
-        1a1. ExpenseExpert requests to check the budget amount parsed and enter command again.
-        Use case ends.
-
-    2a. Missing syntax or syntax field(s).
-        2a1. ExpenseExpert request to check command parsed and enter command again.
-        Use case ends.
-
-    3a. Entered budget is empty
-        3a1. ExpenseExpert notifies the user of the invalidity of the empty budget.
+    2a. Expense Expert detects errors in details of monthly budget.
+        2a1. ExpenseExpert requests to check details of monthly budget parsed and enter command again.
         Use case ends.
 
 **Use case: UC08 Filter expense(s)**
 
 **MSS**
 1. User requests to filter the expense list.
-2. Expense Expert sets a monthly budget for the user.
+2. User supplies filter criteria.
+3. Expense Expert sets a monthly budget for the user.
```

Comment:
Why does Expense Expert set a monthly budget under the use case "Filter expense(s)"?